### PR TITLE
Center EARTH_AGE map at long 0

### PR DESCRIPTION
**Pull request by Esteban82**
I modify the script (and the map) to center at long 0 (instead of 180).

### CODE REVIEW

File: scripts/GMT_age.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Makes crustal age map for docs/earth-age.rst
 gmt begin GMT_age png
 	gmt set MAP_TICK_LENGTH 2p MAP_ANNOT_OFFSET 2p MAP_FRAME_PEN 0.5p GMT_DATA_SERVER candidate
-	gmt grdimage @earth_age_06m -JQ180/6i -C@earth_age.cpt -B0 --MAP_FRAME_PEN=1p
```
Comment (seisman) [inline scripts/GMT_age.sh:5]: Use a lower-resolution grid to have a smaller image size?

Comment (Esteban82) [reply scripts/GMT_age.sh:5]: Fine by me. If I use `@earth_age` instead, them the 10m resolution is used and the png is 112K. 

Paul?

**Comment by Esteban82** [comment]
In the last commit I use the 10m instead.

**Comment by seisman** [comment]
> In the last commit I use the 10m instead.

It's 112 KB, and the workflow compressed it to 74 KB.

File: scripts/GMT_age.sh (hunk 1 above)
Comment (PaulWessel) [reply scripts/GMT_age.sh:5]: 10m is fine with me

**Review by PaulWessel — APPROVED** [review]

**Pull request merged**